test/powershell/Language/Parser/Parser.Tests.ps1

```diff
@@ -182,6 +182,26 @@ Describe "ParserTests (admin\monad\tests\monad\src\engine\core\ParserTests.cs)"
         $result | Should -Be 1
     }
 
+    It "Test that hex named script executed without tabcompleted prefix will run" {
```

Comment:
These two tests are almost the same, you should change it to use `-testcases`, it'll look something like:

```powershell
It 'Test that hex named script executes: <name>' -TestCases @(
  @{ name = '0x.ps1' }
  @{ name = '-0x.ps1' }
  @{ name = '+0x.ps1' }
){
  param ($name)

...
}
```

Reply:
Thanks! I've pushed those changes. Do you have any idea as to the failures on Windows? [Linked in this comment](https://github.com/PowerShell/PowerShell/pull/18349#issuecomment-1287932303)

I can't get the script to execute on MacOS or Linux (even though I can get them to run line by line on Linux when testing manually) so i'm considering muting the tests for everything apart from Windows, but I really need to figure out the reason for those Windows cases failing.

Reply:
Those cases make it hard to execute `-0x` or `+0x`.  the parm is interpreted as a string unless I execute it with the powershell prefix, but for those cases it also doesn't execute. 
![image](https://user-images.githubusercontent.com/19718175/197697342-765676f7-27c4-4e84-99c1-f6ee72a11bc7.png)

Reply:
Hmm. I think you can workaround it with `& ([scriptblock]::Create($name))` possibly?

Reply:
I ended up going with that after suggestions from Chris Dent in the Discord Server. Of course now I have a whole other deluge of test failures...